diff --git a/build/seriously.js b/build/seriously.js
index 88e59c35588e19850d2b60e3071178057b8ab076..55e0af62b0d19d4ba515fad907a457f1577159f6 100644
--- a/build/seriously.js
+++ b/build/seriously.js
@@ -3736,6 +3736,8 @@
 				width = target.width;
 				height = target.height;
 
+				console.log("target", target.getContext('webgl2'));
+
 				//try to get a webgl context.
 				if (!gl || gl.canvas !== target && opts.allowSecondaryWebGL) {
 					context = getWebGlContext(target, {
@@ -3745,6 +3747,16 @@
 						stencil: true,
 						debugContext: debugContext
 					});
+				} 
+				
+				if (!context) {
+					context = target.getContext('webgl2', {
+						alpha: true,
+						premultipliedAlpha: true,
+						preserveDrawingBuffer: true,
+						stencil: true,
+						debugContext: debugContext
+					})
 				}
 
 				if (!context) {
diff --git a/build/seriously.module.js b/build/seriously.module.js
index 0556842f3200b3a05117aa04e6514d687ebe4765..97e068b48e556e6204cc1b89563bfdffc8877003 100644
--- a/build/seriously.module.js
+++ b/build/seriously.module.js
@@ -3733,6 +3733,8 @@ function Seriously(options) {
 			width = target.width;
 			height = target.height;
 
+			console.log("target", target.getContext('webgl2'));
+
 			//try to get a webgl context.
 			if (!gl || gl.canvas !== target && opts.allowSecondaryWebGL) {
 				context = getWebGlContext(target, {
@@ -3744,6 +3746,16 @@ function Seriously(options) {
 				});
 			}
 
+			if (!context) {
+				context = target.getContext('webgl2', {
+					alpha: true,
+					premultipliedAlpha: true,
+					preserveDrawingBuffer: true,
+					stencil: true,
+					debugContext: debugContext
+				})
+			}
+
 			if (!context) {
 				if (!opts.allowSecondaryWebGL && gl && gl.canvas !== target) {
 					throw new Error('Only one WebGL target canvas allowed. Set allowSecondaryWebGL option to create secondary context.');
